lib/components/link.vue

```diff
@@ -13,8 +13,7 @@
        :event="event"
        :tag="tag"
        :class="linkClassObject"
-       @click="linkClick"
-    >
+       @click="linkClick">
```

Comment:
I'm having a hard time testing the `click.native` in the playground. The playground doesn't get the $router. But I'm willing to go on faith here 😉